### DISCUSSION TITLE
无法用raidrive挂载

### DISCUSSION THREAD

**Discussion by ironstraight** [AlistGo/alist#1431] [Q&A]
alist仅在电脑上安装挂载，没在VPS或者软路由搭建。   本地配合raidrive挂载到本地时提示授权身份验证失败。提示不是webdave的url链接。

![屏幕截图 2022-07-30 223951](https://user-images.githubusercontent.com/100616725/181920072-0faab285-2cab-4a51-bebe-94d44a6c699b.jpg)
![屏幕截图 2022-07-30 224036](https://user-images.githubusercontent.com/100616725/181920079-f6caffb3-5b61-4555-a5ea-a7441131ad5b.jpg)

[the remark this answers is not in the document]
**Reply by ironstraight** [reply]
> /dav

有没有/dav 一样显示这样，挂载失败。已经都试过了。

[the remark this answers is not in the document]
**Reply by rakaki** [reply]
很有可能是你密码填错了。注意webdav的密码和网页管理的密码不一样。dav必须填

[the remark this answers is not in the document]
**Reply by ironstraight** [reply]
> 填

密码已经重复确认10次了 也重置过，也试过另外设置密码和账号，全部没用。知道管理密码和 webdav密码是不同的。  所有的都试过了。没用都同样提示无法以上附图的认证失败，我留意英文的提示是说反馈的不是webdav url,而是一个html页面，这是什么意思不懂。

[the remark this answers is not in the document]
**Reply by rakaki** [reply]
alist是在本机上部署的吗

[the remark this answers is not in the document]
**Reply by ironstraight** [reply]
> alist是在本机上部署的吗

是的 本机部署。

**Comment by madiscko** [comment]
/dav 加到这里
![image](https://user-images.githubusercontent.com/30195932/189286701-b5947eb1-acb8-4f8f-aa00-2e9e65747393.png)

**Reply by ironstraight** [reply]
我重装了系统可以了。原来有没有/dav都不行。

**Comment by uake** [comment]
问题解决了吗？我的问题和你一模一样！如果解决了，希望能给我讲讲

**Reply by ironstraight** [reply]
我重装了系统可以了。原来有没有/dav都不行。

**Comment by freeflyfree** [comment]
我同样的问题，加上 /dav   就可以了

**Reply by slsher** [reply]
我也可以了 加上/dav

**Comment by Zhangdata** [comment]
![86dd9db2-6a7a-4d35-b060-6f2a54a10eee](https://github.com/user-attachments/assets/86c6ba07-ce26-48bb-8f58-7fd0254bb78d)
这个问题怎么解决？

**Comment by oOJohn6Oo** [comment]
我尝试挂载内网服务到 Infuse，账户名称使用 guest 或者不填就不行，使用 admin 加密码也不行，然后我到 Alist 管理 - 用户 - admin - 编辑，加上 WebDav 读取就可以了。
